assets/content/less/buttons/buttons.less

```diff
@@ -0,0 +1,73 @@
+// lesshint qualifyingElement: { enabled: true, allowWithClass: true, allowWithAttribute: true }
+.btn {
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-size: 14px;
+    padding-left: 29px;
+    padding-right: 31px;
+
+    &:hover {
+        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.23);
+    }
+
+    // Blue button
+    &.btn-primary,
+    &.btn-primary:hover {
+        background: #65a6de;
```

Comment:
Oh i didnt see it in bower. Yes, in the end it is improvements to make the best out of Bootstrap, such as being able to modify variables only, include bootstrap classes inside a custom class, or use mixins, etc.

So it makes sense if you wanna do it as part of an improvement round 👍